R/Grid.R

```diff
@@ -163,3 +182,24 @@ setValidity(
 length.QuantityCollapser <- function(x) {
     length(x@indexes)
 }
+
+
+expand_null_subs_to_all_subs <- function(x, data, ...) {
```

Comment:
would be better to have minimal documentation and tests for this internal function